Dockerfile

```diff
@@ -13,35 +13,38 @@
 ARG SOURCE_BRANCH=master
-ENV SOURCE_BRANCH="$SOURCE_BRANCH" GOROOT=/usr/lib/go GOPATH=/gopath GOBIN=/gopath/bin AKAMAI_CLI_HOME=/cli
-RUN mkdir /cli && \
+ARG AKAMAI_CLI_PACKAGE_REPO=https://developer.akamai.com/cli/package-list.json
+ENV SOURCE_BRANCH="$SOURCE_BRANCH" GOROOT=/usr/lib/go GOPATH=/gopath GOBIN=/gopath/bin AKAMAI_CLI_HOME=/cli AKAMAI_CLI_PACKAGE_REPO="$AKAMAI_CLI_PACKAGE_REPO"
+RUN mkdir -p /cli/.akamai-cli && \
     if [[ $SOURCE_BRANCH == "master" ]]; then \
-        apk add --no-cache git python2 python2-dev py2-pip python3 python3-dev wget jq openssl openssl-dev  curl nodejs build-base libffi libffi-dev go npm && \
+        apk add --no-cache python2 python3 openssl nodejs libffi go && \
+        apk add --no-cache -t .build-deps git python2-dev py2-pip python3-dev jq openssl-dev curl build-base libffi-dev npm && \
         export PATH=$PATH:$GOROOT/bin:$GOPATH/bin && \
         mkdir -p $GOBIN && \
-        curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && \
+        curl -s https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && \
         go get github.com/akamai/cli && \
         cd $GOPATH/src/github.com/akamai/cli && \
         dep ensure && \
-        go build -o akamai-master-linuxamd64; \
+        go build -o /usr/local/bin/akamai; \
     else \
-        apk add --no-cache git python2 python2-dev py2-pip python3 python3-dev wget jq openssl openssl-dev  curl nodejs build-base libffi libffi-dev npm && \
-        wget `curl https://api.github.com/repos/akamai/cli/releases/latest | jq .assets[].browser_download_url | grep linuxamd64 | grep -v sig | sed s/\"//g`; \
+        apk add --no-cache python2 python3 openssl nodejs libffi go && \
+        apk add --no-cache -t .build-deps git python2-dev py2-pip python3-dev jq openssl-dev curl build-base libffi-dev npm && \
+        curl -s -o /usr/local/bin/akamai `curl https://api.github.com/repos/akamai/cli/releases/latest | jq -r .assets[].browser_download_url | grep linuxamd64 | grep -v sig`; \
     fi && \
-    mv akamai-*-linuxamd64 /usr/local/bin/akamai && chmod +x /usr/local/bin/akamai && \
-    mkdir -p /cli/.akamai-cli && \
-    curl https://developer.akamai.com/cli/package-list.json | jq .packages[].name | sed s/\"//g | xargs akamai install --force
+    pip2 install --no-cache-dir --upgrade pip && \
+    pip3 install --no-cache-dir --upgrade pip && \
+    curl -s "$AKAMAI_CLI_PACKAGE_REPO" | jq -r .packages[].name | xargs akamai install --force && \
+    apk del .build-deps
 
 RUN echo "[cli]" > /cli/.akamai-cli/config && \
     echo "cache-path            = /cli/.akamai-cli/cache" >> /cli/.akamai-cli/config && \
     echo "config-version        = 1" >> /cli/.akamai-cli/config && \
     echo "enable-cli-statistics = false" >> /cli/.akamai-cli/config && \
-    echo "last-ping             = 2018-04-27T18:16:12Z" >> /cli/.akamai-cli/config && \
+    echo "last-ping             = $(date --utc +%FT%TZ)" >> /cli/.akamai-cli/config && \
```

Comment:
The value isn't used for anything, because `enable-cli-statistics` is `false`. I'm not opposing this change, but just wanted to note this.

Reply:
Ok, then we should completely remove this line ... what do you think ?